crawl-ref/source/geom2d.h

```diff
@@ -12,13 +12,13 @@ struct vector
         : x(_x), y(_y) {}
 
     const vector& operator+=(const vector &v);
-    vector operator+(const vector &v) const PURE;
-    vector operator-() const PURE;
+    vector operator+(const vector &v) const;
+    vector operator-() const;
     const vector& operator-=(const vector &v);
-    vector operator-(const vector &v) const PURE;
+    vector operator-(const vector &v) const;
 };
 
-vector operator*(double t, const vector &v) PURE;
+vector operator*(double t, const vector &v);
```

Comment:
I'm sad to be losing these PURE annotations just to appease a compiler that doesn't support them. Can we leave them in, and instead redefine the PURE macro to expand to nothing while under MSVC?